docker-compose.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+
+services:
+  firefly-db:
+    volumes:
+      - firefly-dev-storage:/var/lib/mysql
+      - .:/var/www/firefly-iii
```

Comment:
Just for dev, as a separate task like the migration one. The other option would be to mount the folder individually sounds really weird to me.